doc/error-handling/error-handling.rst

```diff
@@ -342,3 +353,35 @@ If the error code is 6088, it might be easier to bypass this error code first, a
 See :ref:`lsst.ts.m2gui-user_alarm_warn` for the details.
 But the action to bypass the error code might break the system totally.
 Therefore, it would be better to check with the maintainers first before bypassing the error code.
+
+.. _lsst.ts.m2gui-error_power_cycle:
+
+Power-Cycle
+===========
+
+If the M2 is on the TMA, you might need to use the internet interface of general power distribution unit (PDU) to power-cycle it.
+Before the power-cycle, you should check with the hardware or system engineers first.
+The internet interface of PDU is `tea_pdu <https://tea-pdu01.cp.lsst.org/>`_ and the related credential is in `1Passoword MainTel Vault <https://lsstit.1password.com/signin>`_.
+You can only reach this interface when you are under the control network, and the VPN can not reach it directly.
+You can power off followed by powering on the M2 cabinet (wait >30 seconds in between) in the following widget:
+
+.. figure:: ../screenshot/pdu_m2.png
+  :width: 550
+
+  Power-cycle the M2 on the general PDU.
+
+Otherwise, if the M2 is on the level 3, it should be straigt-forward to unplug and replug the power cable (wait >30 seconds in between).
+
+.. _lsst.ts.m2gui-error_reset_m2_interlock_signal:
+
+Reset the M2 Interlock Signal
+=============================
+
+To reset the M2 interlock signal, if the M2 is on the TMA, you might need to push the **RESET** blue button of the **M2 ACTUATOR** tab on the global interlock system (GIS) cabinet at the level 2 as the following:
```

Comment:
Add: after having pressed the reset button, wait 2 minutes before restoring the system with the M2 Python GUI

Reply:
Done in the commit: [7afbfa6](https://github.com/lsst-ts/ts_m2gui/pull/56/commits/7afbfa66dbd5c9388111d725993e9081f10114df). Thanks!